src/ui/ImageGrid/index.scss

```diff
@@ -7,7 +7,6 @@
 
 .sendbird-image-grid {
   border-radius: 12px;
-  direction: rtl;
```

Comment:
Not sure about this change

Reply:
right to left read is not what we want and this is grid limitation. So I asked Gabie to change to 'lfr'